src/processes/boundingBox.js

```diff
@@ -0,0 +1,41 @@
+module.exports = room => {
+  if (!room) {
+    const Room = require('@living-room/client-js')
+    room = new Room()
+  }
+
+  const box = [0.25, 0.25, 0.5, 0.5]
```

Comment:
Changing this to 
```js
const box = { x: 0.25, y: 0.25, w: 0.5, h: 0.5 }
```
I think will make it easier to read the assert code below

Reply:
In addition, maybe doing a subscription to change the box from within the room, like
```js
room.subscribe(`set the animal bounding box to $x, $y, $w, $h`, newBox => box = newBox)
```

Reply:
Also how do you feel about defaults like 0, 0, 1, 1 so we can use the whole table?